tasks/client_config_dir.yml

```diff
@@ -0,0 +1,5 @@
+- name: Write client configs to client-config-dir
+  template:
+    src: staticclient.j2
+    dest: "{{ openvpn_base_dir }}/staticclients/OpenVPN-Client-{{  inventory_hostname[:24]  }}-{{  item.name[:24]  }}"
+  when: item.ip_address is defined
```

Comment:
How does this "allow unconfigured clients"? I don't quite understand your commit message.
As I see it, only _configured_ clients with defined IP addresses are known by the server?

Reply:
You can either have a client with a defined ip address or otherwise openvpn uses dhcp so what i meant was that you can now configure both

Reply:
@SlothOfAnarchy pls rereview